future/name-resolution/recursive-dns/README.md

```diff
@@ -625,7 +625,135 @@ NOTE: We hacked this together a little... We used glue records in the zonefile w
+21:11:07.561667 ARP, Reply 1.2.0.100 is-at 02:42:01:02:00:64, length 28
+```
+
+A standard ARP request. If you look at your network map, you'll see that `resolver-c` has IP Address `1.2.0.100`. So `1.2.0.3` (`router-c3`) is asking which machine on this network is `1.2.0.100`. The next line is `resolver-c` replying to `router-c3`'s ARP request with it's own MAC address. For more on this, checkout the [IP and MAC addresses appendix](../../../appendix/ip-and-mac-addresses.md)
+
+```bash
+21:11:07.561754 IP 1.1.0.200.48600 > 1.2.0.100.53: 48085+ [1au] A? www.awesomecat.com. (59)
+```
+
+<!-- TODO: maybe define A record beyond just the paranthetical? Or do an aside on record types?  -->
+`1.1.0.200` (`client-c1`) sends a request to `1.2.0.100` (`resolver-c`) port `53` requesting the `A` records (the IPv4 records) for `www.awesomecat.com`.
+
+```bash
+21:11:07.562914 IP 1.2.0.100.47301 > 101.0.1.100.53: 35603% [1au] NS? . (28)
+21:11:07.564677 IP 101.0.1.100.53 > 1.2.0.100.47301: 35603*- 2/0/3 NS rootdns-i.isc.org., NS rootdns-n.netnod.org. (123)
+```
+
+<!-- TODO: define NS/SOA records -->
+`1.2.0.100` (`resolver-c`) sends a request (`>`) to `101.0.1.100` (`rootdns-n`) for the `NS` records for `.`, the root of all DNS. Then, `101.0.1.100` (`rootdns-n`) sends a reply back to `1.2.0.100` (`resolver-c`) providing the `NS` records for the root DNS servers.
+
+```bash
+21:11:07.565703 IP 1.2.0.100.23565 > 100.0.1.100.53: 29379% [1au] A? com. (32)
+21:11:07.566389 IP 1.2.0.100.24580 > 101.0.1.100.53: 15555% [1au] A? org. (32)
+21:11:07.566667 IP 1.2.0.100.42017 > 100.0.1.100.53: 52879% [1au] A? org. (32)
+21:11:07.566698 IP 100.0.1.100.53 > 1.2.0.100.23565: 29379- 0/1/2 (78)
+21:11:07.566861 IP 101.0.1.100.53 > 1.2.0.100.24580: 15555- 0/1/2 (78)
+21:11:07.567274 IP 100.0.1.100.53 > 1.2.0.100.42017: 52879- 0/1/2 (78)
+```
+
+Now that `resolver-c` has confirmed where the root DNS servers are, It starts firing off requests to learn about all of the TLDs it needs to know about. It has a request from `client-c1` for a name in the `com` TLD, and it just got a response back for root server names in the `org` TLD. So we see 3 requests fired off here, each of them for `A` records for TLDs. The resolver will try to get the fastest possible response for the client. So it's spitting out requests to both of the root DNS servers here to see which response comes back first. 
+
+Interestingly, it doesn't do the same for `com`. This is likely because we didn't clear the cache on our whole internet. If we were to restart each DNS server on our internet, we'd likely see requests for `com` to both root DNS servers.
+
+The next 3 lines are the responses back from the root DNS servers. The actual response bodies aren't parsed here, but given what we saw in the exercises above, we know that what should be seeing DNS responses for where the `com` and `org` TLD servers are. These responses will include the IP addresses for those servers.
```

Comment:
> The actual response bodies aren't parsed here...

I believe we can get much more detailed data on these interactions if we turned up verbosity on `tcpdump` (e.g. `tcpdump -nvv`)

```suggestion
The next 3 lines are the responses back from the top-level domain DNS servers. The actual response bodies aren't parsed here, but given what we saw in the exercises above, we know that what should be seeing DNS responses for where the `com` and `org` TLD servers are. These responses will include the IP addresses for those servers.
```